<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="read">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>read SIS data file</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="readfolder">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>read SIS data folder</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sis3302">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3302</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sis3302wrap">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3302  Wrap</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sis3320">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3320</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="calFlag">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate Energy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="norawFlag">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Data Only</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
